### PR TITLE
UI for editing and searchbars.

### PR DESCRIPTION
**Pull request by RaimondSob**
Added search bars for drinks and bars, edit page now has 3 buttons(edit price, add bar, add drink), those do not work yet(still figuring the flyout and routing), added 3 new pages for those buttons. 
My initial though was to have Radio Buttons for editing, but apparently those are in experimental state in xamarin, so was a bit afraid to use them and decided to use buttons that send you to new page, maybe it is even easier that way.
There are no buttons for save, but there is no backend for that, so creating those buttons later will be no problem.

### CODE REVIEW

**Comment by ElektrosStulpas** [comment]
What do you mean there's no backed for saving?

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddBarPage.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (ElektrosStulpas) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddBarPage.xaml.cs:6]: empty space

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        TitleColor="Red">
+                <Picker.ItemsSource>
+                    <x:Array Type="{x:Type x:String}">
+                        <x:String>Baboon</x:String>
+                        <x:String>Capuchin Monkey</x:String>
+                        <x:String>Blue Monkey</x:String>
+                        <x:String>Squirrel Monkey</x:String>
+                        <x:String>Golden Lion Tamarin</x:String>
+                        <x:String>Howler Monkey</x:String>
+                        <x:String>Japanese Macaque</x:String>
```
Comment (ElektrosStulpas) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml:17]: when did bars turn into monkeys :D

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (ElektrosStulpas) [inline WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml.cs:6]: empty whitespace

**Review by ElektrosStulpas — COMMENTED** [review]
that's a lot of monkeys man

File: WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml (hunk 2 above)
Comment (RaimondSob) [reply WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml:17]: an example list to see how it would look like on phone

**Review by RaimondSob — COMMENTED** [review]
No backend for saving I just meant methods, my bad.

**Pull request merged**